requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.22.4
+numpy==1.26.4
```

Comment:
@samszotkowski This version was required for me. Can you see if everything still works correctly for you as well?

Reply:
@allenkinzalow yep this worked for me